modelica_builder/model.py

```diff
@@ -165,6 +166,34 @@ def update_component_argument(self, type_, identifier, argument_name, new_value,
 
         self.add(Transformation(selector, Edit.make_replace(new_value)))
 
+    def add_parameter(self, type_, identifier, arguments=None, assigned_value=None, string_comment=None, annotations=None):
+        """add_parameter inserts a new parameter at the top of the model's element list
+
+        :param type_: string, type of the component
```

Comment:
i don't think i've seen the `type_` convention. Is this to prevent using the `type` keyword?

Reply:
Looks like I will have to rename klass to class_ in the GMT